cc/oak_echo_raw_enclave_app/main.cc

```diff
@@ -19,7 +19,8 @@
 
 constexpr int CHANNEL_FD = 10;
 
-int main(int argc, char* argv[]) {
+// Under C++ rules, the name is mangled to "_Z4mainiPPc" -- find out what's going on with that.
```

Comment:
It may be that it needs to be marked as `[[gnu::used]]` or something like that, but not sure.